docs/source/Support/bskReleaseNotes.rst

```diff
@@ -61,6 +61,7 @@ Version |release|
 - Support calling ``unsubscribe`` on input messages.
 - Fixed an issue where the :ref:`forceTorqueThrForceMapping` module's Reset() function did not zero all thruster settings correctly.
 - Updated ``canary`` workflow to run on all pull requests to the develop branch, providing feedback on compatibility with latest dependencies.
+- Converted RW data structures in ``reactionWheelStateEffector`` to shared pointers instead of raw pointers.
```

Comment:
add a bullet to the known issues RST file that the way to create the RW configuration has moved aware from `messages`. If the user is using the RW factory class no code changes are needed.  If they are creating this on their own then then show them what needs to be changed.